src/request/components/request-detail-panel-execution.jsx

```diff
@@ -52,21 +54,54 @@ var getMessages = (function() {
     }
 })();
 
+var combineMongoMessages = function(message) {
```

Comment:
Can we use `let` or `const` instead, along with arrow functions and other ES2015 enhancements?